themes/default/content/templates/static-website/gcp/index.md

```diff
@@ -22,7 +24,7 @@ The Google Cloud Static Website template creates an infrastructure as code proje
 
 ![An architecture diagram of the Pulumi Google Cloud Static Website template](./architecture.png)
 
-## Using this template
+## Using this template to host a Google Cloud Static Website
```

Comment:
Same as previous (I'd prefer "Host a Static Website on Google Cloud"), if that will still get the same SEO optimizations

Reply:
Keep in mind changes like these have a negative impact the right nav:

![image](https://user-images.githubusercontent.com/274700/207677807-c3822aee-847e-47f7-bf6f-188d3eb43360.png)

Will take a closer look at these changes today, but in general would prefer to be conservative and strike a balance between SEO and usability/consistency.